script.colorbox/addon.xml

```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.colorbox" name="colorbox Script" version="2.0.1" provider-name="BADMS">
+<addon id="script.colorbox" name="colorbox Script" version="2.0.2" provider-name="BADMS">
 	<requires>
 		<import addon="xbmc.python" version="2.25.0"/>
 		<import addon="script.module.simplejson" version="2.0.10"/>
 		<import addon="script.module.pil" version="1.1.7"/>
 	</requires>
-    <extension point="xbmc.python.script"           library="default.py">
+    <extension point="xbmc.python.script" library="default.py">
         <provides>executable</provides>
     </extension>
 	<extension point="xbmc.addon.metadata">
+	    <summary lang="en">Image manipulator and colour matching tools</summary>
 		<language/>
```

Comment:
please remove the language tag (only applied to content plugins - video, audio, etc)